generator/video/video_generator.go

```diff
@@ -35,7 +35,7 @@ func ScaleImages(images []os.DirEntry, path string) {
 func GenerateVideo(path string, imageExt string) error {
 	imageInput := ffmpeg.Input(path+PREFIX+"%03d."+imageExt, ffmpeg.KwArgs{"loop": 1, "framerate": "1/2"})
 
-	audioInput := ffmpeg.Input("./data/test_data/audio.mp3")
```

Comment:
Please update the code to be able to run on the test data as well or move the audio file to the specified path to ensure that the program can still run by itself. You can take inspiration from how the images are handled

Reply:
test_audio is now in use if audio not supplied! :tada: